src/canister/individual_user_template/src/data_model/memory.rs

```diff
@@ -16,6 +16,8 @@ const POST_PRINCIPAL_MEMORY: MemoryId = MemoryId::new(3);
 const SLOT_DETAILS_MEMORY: MemoryId = MemoryId::new(4);
 const KV_STORAGE_NAMESPACE_MEMORY: MemoryId = MemoryId::new(5);
 const KV_STORAGE_NAMESPACE_KEY_VALUE_MEMORY: MemoryId = MemoryId::new(6);
+const WATCH_HISTORY_MEMORY: MemoryId = MemoryId::new(9);
+const SUCCESS_HISTORY_MEMORY: MemoryId = MemoryId::new(10);
```

Comment:
Any reason we are memory id 7 and 8?

Reply:
@abeeshake-yral is using 7 and 8. Didn't want to conflict

Reply:
changed